workspaces/refresh_liferay_sample_workspace.sh

```diff
@@ -227,6 +227,126 @@ EOF
 	../tools/create_custom_element.sh liferay-sample-custom-element-3 angular
 
 	mv liferay-sample-custom-element-3 liferay-sample-workspace/client-extensions
+
```

Comment:
So, what's this doing? :flushed: 

If the files are added explicitly to the sample workspace, why is this script needed and for what? :thinking: